webhook/webhook.go

```diff
@@ -298,13 +298,21 @@ func (ac *AdmissionController) Run(stop <-chan struct{}) error {
 		return nil
 	}
 
+	serverBootstrapErrCh := make(chan struct{})
 	go func() {
 		if err := server.ListenAndServeTLS("", ""); err != nil {
 			logger.Error("ListenAndServeTLS for admission webhook returned error", zap.Error(err))
+			serverBootstrapErrCh <- struct{}{}
```

Comment:
Close the channel here instead of sending a message, since the message value is never used.

```suggestion
			close(serverBootstrapErrCh)
```

Reply:
I agree, closing is better here.